src/azul/indexer/__init__.py

```diff
@@ -200,6 +200,17 @@ def __str__(self) -> str:
     def effective(self) -> SOURCE_SPEC:
         return attr.evolve(self, prefix=self.prefix.effective)
 
+    def contains(self, other: 'SourceSpec') -> bool:
+        """
+        >>> wide = SimpleSourceSpec.parse('foo:4')
+        >>> narrow = SimpleSourceSpec.parse('foo:42')
+        >>> wide.contains(narrow)
+        True
+        >>> narrow.contains(wide)
```

Comment:
```suggestion

        >>> narrow.contains(wide)
```

Reply:
Add cases for a source containing itself and for two sources that don't contain each other.